src/apps/is-app-in-tunnel.ts

```diff
@@ -0,0 +1,4 @@
+const tunnelKeywords = [".ngrok.io", ".saleor.live"];
+
+export const isAppInTunnel = (manifestUrl: string) =>
+  Boolean(tunnelKeywords.find(keyword => manifestUrl.includes(keyword)));
```

Comment:
This crashes in current usage. Either something like this:
```suggestion
export const isAppInTunnel = (manifestUrl: string | undefined) =>
  Boolean(tunnelKeywords.find(keyword => manifestUrl?.includes(keyword)));
```
or use fallback string in current usage.